src/docs/arch-transaction.dox

```diff
@@ -1,8 +1,202 @@
 /*! @arch_page arch-transaction Transactions
 
-Transactions provide a powerful abstraction for multiple threads to operate on data concurrently.
-A caller of WiredTiger uses @ref transactions within the API to start and stop transactions within
-a session (thread of control).
+Transactions provide a powerful abstraction for multiple threads to operate on data
+ concurrently. A caller of WiredTiger uses @ref transactions within the API to start and stop
+ transactions within a session (thread of control).
 
 Internally, the current transaction state is represented by the WT_TXN structure.
+
+WiredTiger performs all the operations in transactions. If the user doesn't explicitly begin a
```

Comment:
This reads weirdly, "WiredTiger performs all operations within a transaction". That isn't true for operations like schema drop, create etc. How about "WiredTiger peforms most operations within a transaction" or similar?